sensu/plugins/check-l3-router.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+HOSTNAME=`hostname`
+for IFACE in $(ifquery --list); do
```

Comment:
Should be able to explicitly check eth0 and eth0.400, and make sure that both have an ip, right?